wrangler-storage/src/main/java/co/cask/wrangler/dataset/connections/ConnectionStore.java

```diff
@@ -61,18 +70,44 @@
   private static final Gson GSON = new GsonBuilder()
     .registerTypeAdapter(Schema.class, new SchemaTypeAdapter()).create();
   private static final Type MAP_TYPE = new TypeToken<Map<String, String>>() { }.getType();
+  private static final String NAMESPACE_COL = "namespace";
+  private static final String ID_COL = "id";
   private static final String TYPE_COL = "type";
   private static final String NAME_COL = "name";
   private static final String DESC_COL = "description";
   private static final String PROPERTIES_COL = "properties";
   private static final String CREATED_COL = "created";
   private static final String UPDATED_COL = "updated";
-  private final Table table;
+  private static final StructuredTableId TABLE_ID = new StructuredTableId("connections");
```

Comment:
should we also change javadoc for this store?

Reply:
added a couple lines about the schema and primary key.